spec/spec_helper.rb

```diff
@@ -9,7 +9,12 @@
 
 Dotenv.load
 SimpleCov.start
-SimpleCov.formatter = Coveralls::SimpleCov::Formatter
+
+SimpleCov.formatter = if ENV.fetch("local_simplecov") == "true"
+                        SimpleCov::Formatter::HTMLFormatter
+                      else
+                        Coveralls::SimpleCov::Formatter
+                      end
```

Comment:
What do you think about using multiple formatters?

```suggestion
SimpleCov.formatters = [
  SimpleCov::Formatter::HTMLFormatter,
  Coveralls::SimpleCov::Formatter
]
```